src/app/core/admin/admin-entity-details/admin-entity-form/admin-entity-form.component.html

```diff
@@ -36,13 +36,24 @@
             Edit Field
           </button>
 
-          <div class="dummy-form-field">
+          <div class="dummy-form-field flex-row align align-center">
             <app-entity-field-edit
               [field]="field"
               [entity]="dummyEntity"
               [form]="dummyForm"
             ></app-entity-field-edit>
           </div>
+
+          <button
+            mat-icon-button
+            color="black"
+            class="field-hide-button"
+            (click)="hideField(field, group)"
+            aria-label="Hide Field"
+            matTooltip="Click to hide this field from the form"
```

Comment:
```suggestion
            matTooltip="remove (i.e. hide) this field from the form"
            i18n-matTooltip
```

Reply:
`i18n-matTooltip` enables that text for translation into different interface languages